src/components/SearchTracePage/index.js

```diff
@@ -218,20 +220,32 @@ SearchTracePage.propTypes = {
   errorMessage: PropTypes.string,
 };
 
+const stateTraceXformer = getLastXformCacher(stateTrace => {
+  const { traces: traceMap, loading, error: traceError } = stateTrace.toJS();
```

Comment:
The state is an [Immutable.js](https://facebook.github.io/immutable-js/) [Map](https://facebook.github.io/immutable-js/docs/#/Map). [`.toJS()`](https://facebook.github.io/immutable-js/docs/#/Map/toJS) converts it to a plain JS object. The `.toJS()` call is probably superfluous; I'll go over why I didn't take it out in our discussion Wednesday.